File: code_conversion_js/dog.mjs
import { Animal } from "./animal.mjs";
export { Dog };
class Dog extends Animal
{
    #breed;

    constructor(name, hasFur, numberOfLegs, sizeInFeet, breed)
    {
        super(name, hasFur, numberOfLegs, sizeInFeet);
        this.#breed = breed;
        Object.seal(this);
    }

    getDescription = () => super.getDescription() + ` This dog is a ${this.#breed}.`
}

File: CodeConversionCS/CodeConversionCS/Dog.cs
public class Dog : Animal
{
    private String Breed { get; set; }

    public Dog(String name, bool hasFur, int numberOfLegs, double sizeInFeet, String breed) 
        : base(name, hasFur, numberOfLegs, sizeInFeet)
    {
        this.Breed = breed;
    }

    public override String GetDescription() => base.GetDescription() + $" This dog is a {Breed}.";
}
